resources/extruders/atom2_extruder_0.def.json

```
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "Atom 2",
```

```suggestion
        "machine": "atom2",
```

This needs to match the ID of the machine _exactly_. The ID is defined by the filname (see https://github.com/Ultimaker/Cura/wiki/Adding-new-machine-profiles-to-Cura)